src/common/selectors.ts

```diff
@@ -1,3 +1,6 @@
-export const BODY = "body";
-
-export const SIDEBAR_POSITIONER = "sidebar-positioner";
+export enum SELECTOR {
```

Comment:
Previously, selectors were defined as individual constants. For ease of use and improved maintainability, I've refactored them into an enum. This is a breaking change.